LRRServer/lawreadingrobot/models.py

```diff
@@ -0,0 +1,48 @@
+
+import uuid
+
+from sqlalchemy import JSON, Column, DateTime, Text, Uuid
+from sqlalchemy.sql import func
+
+from .database import Base
+
+
+class BaseMixin(object):
+    created_at = Column(DateTime(timezone=True), server_default=func.now())
+    updated_at = Column(DateTime(timezone=True), onupdate=func.now())
+    # Makes sure the columns are added to the end of the table
+    created_at._creation_order = 9998  # type: ignore
```

Comment:
Optional: There might be better ways to due this using `mapped_column` and/or new behavior from SQLAlchemy 2.0: https://docs.sqlalchemy.org/en/20/changelog/whatsnew_20.html#orm-declarative-applies-column-orders-differently-control-behavior-using-sort-order